include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,481 @@
+    /*!
+     *  enthalpy and entropy are sum of ideal surface species enthalpy and entropy
+     *  and coverage-dependent enthalpy and entropy which are caclulated with
+     *  a piecewise linear function of coverages:
+     *
+     *  \f[ h^{cov}_k(\theta) = \sum_j h^{low}_{k,j} \theta_j
+     *      \text{, for } \theta_j \leq \theta^{change}_{k,j}
+     *  \f]
+     *  \f[ h^{cov}_k(\theta) = \sum_j h^{high}_{k,j}
+     *      (\theta_j - \theta^{change}_{k,j}) +
+     *      (h^{low}_{k,j} \theta^{change}_{k,j}) \text{, for }
+     *      \theta_j > \theta^{change}_{k,j}
+     *  \f]
+     *
+     *  \f[ s^{cov}_k(\theta) = \sum_j s^{low}_{k,j} \theta_j
+     *      \text{, for } \theta_j \leq \theta^\text{change}_{k,j}
+     *  \f]
+     *  \f[ s^{cov}_k(\theta) = \sum_j s^{high}_{k,j}
+     *      (\theta_j - \theta^{change}_{k,j}) +
+     *      (s^{low}_{k,j} \theta^{change}_{k,j}) \text{, for }
+     *      \theta_j > \theta^{change}_{k,j}
+     *  \f]
+     *  @param plin_deps  list of parameters as a PiecewiseDependency object
+     */
+    void setPiecewiseDependency(const PiecewiseDependency& plin_deps);
+
+    //! Set the interpolative coverage dependece for species
+    /*!
+     *  enthalpy and entropy are sum of ideal surface species enthalpy and entropy
+     *  and coverage-dependent enthalpy and entropy which are caclulated with
+     *  a linearly interpolated function of coverages:
+     *
+     *  \f[ h^{cov}_k(\theta) = \sum_j \frac{h^{right}_{k,j}
+     *      - h^{left}_{k,j}}{\theta^{right}_{k,j} - \theta^{left}_{k,j}}
+     *      (\theta_j - \theta^{left}_{k,j}) + h^{left}_{k,j}
+     *      \text{, where } \theta^{left}_{k,j} \leq \theta_j < \theta^{right}_{k,j}
+     *  \f]
+     *
+     *  \f[ s^{cov}_k(\theta) = \sum_j \frac{s^{right}_{k,j}
+     *      - s^{left}_{k,j}}{\theta^{right}_{k,j} - \theta^{left}_{k,j}}
+     *      (\theta_j - \theta^{left}_{k,j}) + s^{left}_{k,j}
+     *      \text{, where } \theta^{left}_{k,j} \leq \theta_j < \theta^{right}_{k,j}
+     *  \f]
+     *
+     *  @param int_deps  list of parameters as an InterpolativeDependency object
+     */
+    void setInterpolativeDependency(const InterpolativeDependency& int_deps);
+
+    //! Set the heat capacity coverage dependece for species
+    /*!
+     *  heat capacity is sum of ideal surface species heat capacity and coverage-dependent
+     *  heat capacity which is caculated using a function with quadratic depedence on coverages
+     *  and a logarithimic dependence on temperature:
+
+     *  \f[ cp^{cov}_k(\theta) = \sum_j (a_{k,j} ln(T) + b_{k,j}) \theta_j^2 \f]
+     *
+     *  @param cpcov_deps  list of parameters as a HeatCapacityDependency object
+     */
+    void setHeatCapacityDependency(const HeatCapacityDependency& cpcov_deps);
+
+    virtual void initThermo();
+    virtual bool addSpecies(shared_ptr<Species> spec);
+
+    // Functions calculating reference state thermodyanmic properties--------------
```

Comment:
If you want to group methods so they shows up that way in the HTML docs, you should write:
```suggestion
    //! @name Functions calculating reference state thermodynamic properties
    //! @{
```

And then after that set of functions, add a comment line containing `//! @}`.